docs/tools/vdb_table/vendor.schema.json

```diff
@@ -36,42 +35,108 @@
     "github_stars": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | GitHub ⭐ | The total number of stars for the core product repository." },
     "docker_pulls": {"allOf": [{"$ref": "#/$defs/statsWithSource"}],"$comment": "Stats | Docker Pulls | The total number of pulls for the core product's docker image." },
     "pypi_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | PyPi Downloads | The number of downloads (in last 90 days) for the core product's python package." },
     "npm_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | NPM Downloads | The number of downloads (in last 90 days) for the core product's NPM package." },
     "crates_io_downloads": {"allOf": [{"$ref": "#/$defs/statsWithSource"}], "$comment": "Stats | Rust Crates Downloads | The number of downloads (in last 90 days) for the core product's Rust crate." }
   },
-
   "$defs": {
     "links": {
       "$id": "links",
       "type": "object",
       "properties": {
-        "docs": {"type": "string"},
-        "github": {"type": "string"},
-        "website": {"type": "string"},
-        "vendor_discussion": {"type": "string"},
-        "poc_github": {"type": "string"},
-        "slug": {"type": "string"}
+        "docs": {
+          "type": "string"
+        },
+        "github": {
+          "type": "string"
+        },
+        "website": {
+          "type": "string"
+        },
+        "vendor_discussion": {
+          "type": "string"
+        },
+        "poc_github": {
+          "type": "string"
+        },
+        "slug": {
+          "type": "string"
+        }
       }
     },
     "featureWithSource": {
       "$id": "featureWithSource",
       "type": "object",
       "properties": {
-        "support": {"enum": ["", "none", "partial", "full"]},
-        "source_url": {"type": "string"},
-        "comment": {"type": "string"}
+        "support": {
+          "enum": [
+            "",
+            "none",
+            "partial",
+            "full"
+          ]
+        },
+        "source_url": {
+          "type": "string"
+        },
+        "comment": {
+          "type": "string"
+        }
       }
     },
     "stringWithSource": {
       "$id": "stringWithSource",
       "type": "object",
       "properties": {
-        "value": {"type": "string"},
-        "source_url": {"type": "string"},
-        "comment": {"type": "string"}
+        "value": {
+          "type": "string"
+        },
+        "source_url": {
+          "type": "string"
+        },
+        "comment": {
+          "type": "string"
+        }
+      }
+    },
+    "indexTypesListWithSource": {
+      "$id": "indexTypesListWithSource",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "FLAT",
```

Comment:
@ClaireSuperlinked maybe we could do a vectorhub article about all of these - would be a fascinating tree of how these evolved from one another and what their benefits are. @dhruv-anand-aintech would you be up for writing about this?

Reply:
Yes I think it would be very interesting. I'm also looking at an article talking about the trends we see with the VDB table in terms of the columns requested, use of Rust vs Python etc - there is a lot of interesting stuff here